### ISSUE TITLE
Error on setbalance

### ISSUE THREAD

**Issue by FabioZumbi12** [Erigitic/TotalEconomy#57]
I having this error on try to set balance with:

/setbalance FabioZumbi12 100

```
[04:37:03] [Server thread/INFO]: Error occurred while executing command: net.minecraft.server.dedicated.DedicatedServer cannot be cast to org.spongepowered.api.entity.living.player.Player
[04:37:03] [Server thread/ERROR] [Sponge]: Error occurred while executing command 'setbalance FabioZumbi12 100' for source DedicatedServer: net.minecraft.server.dedicated.DedicatedServer cannot be cast to org.spongepowered.api.entity.living.player.Player
java.lang.ClassCastException: net.minecraft.server.dedicated.DedicatedServer cannot be cast to org.spongepowered.api.entity.living.player.Player
        at com.erigitic.commands.SetBalanceCommand.execute(SetBalanceCommand.java:31) ~[SetBalanceCommand.class:?]
```
TE version 1.4.0

**Comment by MarkL4YG** [comment]
Hey,

I wasn't able to reproduce the issue, can you provide further information ?

**Comment by MarkL4YG** [comment]
Hey,

First: Please provide more information (maybe next time) as this only seems to happen if you run this command from the console

Then: I'm looking for a fix, but @Erigitic will have to implement a way to handle (or not handle) offline players.

**Comment by MarkL4YG** [comment]
Fix is now included in #55 (PR)
